src/sentry/static/sentry/app/data/forms/sentryApplication.jsx

```diff
@@ -1,119 +1,109 @@
+  },
+  {
+    name: 'webhookUrl',
+    type: 'string',
+    required: true,
+    label: 'Webhook URL',
+    placeholder: 'e.g. https://example.com/sentry/webhook/',
+    help:
+      'The URL Sentry will send requests to for events such as installation changes and any Resource Subscriptions the Integration subscribes to.',
+  },
+  {
+    name: 'redirectUrl',
+    type: 'string',
+    label: 'Redirect URL',
+    placeholder: 'e.g. https://example.com/sentry/setup/',
+    help: 'The URL Sentry will redirect users to after installation.',
+  },
+  {
+    name: 'verifyInstall',
+    label: 'Verify Installation',
+    type: 'boolean',
+    help: 'If enabled, installations will need to be verified before becoming installed.',
+  },
+  {
+    name: 'isAlertable',
+    type: 'boolean',
+    label: 'Alert Rule Action',
+    help: tct(
+      'If enabled, this integration will be an action under alert rules in Sentry. The notification destination is the Webhook URL specified above. More on actions [learn_more:Here].',
       {
-        name: 'schema',
-        type: 'textarea',
-        label: 'Schema',
-        autosize: true,
-        help: 'Schema for your UI components',
-        getValue: val => {
-          return val === '' ? {} : JSON.parse(val);
-        },
-        setValue: val => {
-          const schema = JSON.stringify(val, null, 2);
-          if (schema === '{}') {
-            return '';
-          }
-          return schema;
-        },
-        validate: ({id, form}) => {
-          if (!form.schema) {
-            return [];
-          }
+        learn_more: <a href="https://docs.sentry.io/product/notifications/#actions" />,
+      }
+    ),
+  },
+  {
+    name: 'schema',
+    type: 'textarea',
+    label: 'Schema',
+    autosize: true,
+    help: 'Schema for your UI components',
+    getValue: val => {
+      return val === '' ? {} : JSON.parse(val);
+    },
+    setValue: val => {
+      const schema = JSON.stringify(val, null, 2);
+      if (schema === '{}') {
+        return '';
+      }
+      return schema;
+    },
+    validate: ({id, form}) => {
+      if (!form.schema) {
+        return [];
+      }
 
-          try {
-            JSON.parse(form.schema);
-          } catch (e) {
-            return [[id, 'Invalid JSON']];
-          }
-          return [];
-        },
-      },
-      {
-        name: 'overview',
-        type: 'textarea',
-        label: 'Overview',
-        autosize: true,
-        help: 'Description of your Integration and its functionality.',
-      },
-    ],
+      try {
+        JSON.parse(form.schema);
+      } catch (e) {
+        return [[id, 'Invalid JSON']];
+      }
+      return [];
+    },
+  },
+  {
+    name: 'overview',
+    type: 'textarea',
+    label: 'Overview',
+    autosize: true,
+    help: 'Description of your Integration and its functionality.',
   },
 ];
 
-export default forms;
+export const externalIntegrationForms = [
```

Comment:
This new logic on the bottom is really the only change, the large diff above has to do with storing the list of form fields in a separate variable and using them down below.